frontend/src/app/forms/model/steps.model.ts

```diff
@@ -51,7 +51,7 @@ export const ApplicationStepsConfig: ApplicationStepsConfig = {
   [ApplicationStep.WorkExperience]: {
     useForProgressCalculation: true,
     formItemName: 'workExperienceInfo',
-    maySkipStep: false,
+    maySkipStep: true,
```

Comment:
(This comment is not really related to this line...)

When you skip the Work Experience step, then in the right menu, it will still not be "checked". The reason: There is a bug in `forms/side-menu/side-menu.component.html`. Normally, for every menu item, we define "isDisabled". However, someone (probably me :D ) has forgotten to define this for the Work Experience step: 

![grafik](https://user-images.githubusercontent.com/8061217/87848956-3e821580-c8e5-11ea-8fa2-b4ce4feb044a.png)

So, could you add the missing definition for "isDisabled" to the side menu html? Then, when skipping the Work Experience step, the checkmark should hopefully appear.

Reply:
Yes, this solved the problem :)